### ISSUE TITLE
Template does not update after model change

### ISSUE THREAD

**Issue by Jarzembowski** [bentorfs/angular-bootstrap-multiselect#71]
I'm trying to set the items in the multiselect as _select_ based on a model that I get from an http request.
Im not sure if this has something to do with the order I set the model and the options, but after the promise returns, I got both the options and the model, like this:

```
Model: [
{id: 1, nome: "Cafés"},
{id: 2, nome: "Bares"} ];

Options: [
{id: 1, nome: "Cafés"},
{id: 2, nome: "Bares"},
{id: 3, nome: "Restaurantes"}];

```
And the multiselect like this:

        <multiselect ng-model="Model" options="Options" id-prop="id" display-prop="nome"></multiselect>

The options are displayed correctly, like the options array, but the options on the model are not selected, and I think they should, because the multiselect element is displayed based on both the model and the options. 
Basically the model is updated based on the template, but the opposite is not happening. When I set the model on the controller, I dont see this changes reflected in the template(display the items selected, for example).

![image](https://user-images.githubusercontent.com/18660312/42227374-48a19e58-7ed9-11e8-92e2-29303e0ff256.png)
_Two of the items in this image should be selected, since they are in the model array._

**Comment by labrute** [comment]
I have the same problem. But in my case Options are updated async. 
I have discovered that the Model is reset when i change Options. So i do this :

```
copyModel = angular.copy(Models);
http.get().then((response) => {
  Options = response.data;
  // Here model is reset so i apply the copy
  Models = copyModel;
}
```

But the HTML component is not updated.
Instead of "2 selected" i have "Selection" until i click on drop button.

I try to call manually $digest but it didn't works

**Comment by abastien** [comment]
Same problem here

**Comment by michael-luk** [comment]
Same problem. If the options changed, you have to click the dropdown btn to apply the changes.
